experimental/BasisLieHighestWeight/src/ModuleData.jl

```diff
@@ -0,0 +1,86 @@
+abstract type ModuleData end
+
+# To be implemented by subtypes:
+# Mandatory:
```

Comment:
this list is missing two functions that you expect to exist for subtypes: `L`/`base_lie_algebra` and `highest_weight`